DQMServices/Components/plugins/DQMStoreStats.cc

```diff
@@ -290,6 +290,14 @@ int DQMStoreStats::calcstats(int mode = DQMStoreStats::considerAllME) {
                      getEmptyMetric(it->getTH2D()->GetArray(), it->getNbinsX() + 2, it->getNbinsY() + 2, 0),
                      it->getNbinsX() * it->getNbinsY() * sizeof(double));
         break;
+      case MonitorElement::Kind::TH2I:
+        currentSubfolder.AddBinsI(
+            it->getNbinsX() * it->getNbinsY(),
+            getEmptyMetric(it->getTH2I()->GetArray(), it->getNbinsX() + 2, it->getNbinsY() + 2, 0));
+        curr->update(it->getNbinsX() * it->getNbinsY(),
+                     getEmptyMetric(it->getTH2I()->GetArray(), it->getNbinsX() + 2, it->getNbinsY() + 2, 0),
+                     it->getNbinsX() * it->getNbinsY() * sizeof(double));
```

Comment:
Is there any reason to not set it->getNbinsX() * it->getNbinsY() * sizeof(int));  ?
Thanks

Reply:
thanks. Fixed at [d0f8fd9](https://github.com/cms-sw/cmssw/pull/37665/commits/d0f8fd962d224dc34762f5077e0c5ca136256fdb).